ui/app/src/components/PreviewDialog/PreviewDialogContainer.tsx

```diff
@@ -87,8 +86,7 @@ export function PreviewDialogContainer(props: PreviewDialogContainerProps) {
         closePreviewDialog(),
         showCodeEditorDialog({
           path: url,
-          mode,
-          onClose: batchActions([closeCodeEditorDialog(), conditionallyUnlockItem({ path: url })])
```

Comment:
This actions are not needed since those are already being performer by the code editor dialog